src/app/domain/components/questions/check/check.component.ts

```diff
@@ -33,6 +33,8 @@ export class CheckQuestionsComponent implements OnInit {
     onSubmit() {
         this.questionService.sendQuestions(this.caseId, 1).subscribe(res => {
             this.redirectionService.redirect(`/viewcase/${this.caseId}/questions?sent=success`);
+        }, () => {
+            this.redirectionService.redirect(`/viewcase/${this.caseId}/questions?sent=failure`);
```

Comment:
Should we attempt to log the error here at all? Maybe just a console.log for now?

Reply:
I'm not sure how much useful information actually gets sent down to the front end. It should be logged on the back end for sure.